src/components/directory-content.js

```diff
@@ -160,12 +161,23 @@ const DirectoryContent = () => {
     });
     const [childrenMetadata, setChildrenMetadata] = useState({});
 
-    const [selectedUuids, setSelectedUuids] = useState(new Set());
-
     const currentChildren = useSelector((state) => state.currentChildren);
     const currentChildrenRef = useRef();
     currentChildrenRef.current = currentChildren;
 
+    const currentChildrenUuids = useMemo(
+        () =>
+            currentChildren ? currentChildren.map((e) => e.elementUuid) : [],
```

Comment:
maybe `.filter((e) => !e.uploading)` here to avoid selecting uploading elements.
With this functionality, I think we should disable or add an overlay over the uploading line because the User should know...